src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -1005,6 +1034,30 @@ module.exports = function<T, P, I, TI, C, CX>(config : HostConfig<T, P, I, TI, C
     }
   }
 
+  function scheduleErrorRecovery(fiber : Fiber) {
+    scheduleUpdateAtPriority(fiber, TaskPriority);
+  }
+
+  function scheduleSetState(fiber : Fiber, partialState : any) {
```

Comment:
Can we fine a way to move all of these out of the scheduler? These are super specific to the class API and as we add new APIs we don't want to bloat the scheduler which already has too many unrelated concepts.